lib/db.js

```diff
@@ -389,7 +433,14 @@ class LimitDB {
   }
 
   close(callback) {
-    this._db.close(callback);
+    this._db.close(() => {
```

Comment:
shouldn't be ignore "close" if this._db is not defined (fail to open)

Reply:
good point, updating this.